lib/user_repository.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'user'
+class UserRepository
+
+    # Selecting all records
+    # No arguments
+    def all
```

Comment:
Indenting with 8 chars is way too much, you should only have two space indents in ruby, you should change your vscode settings so that the 'tab' key only inserts two whitespaces, not 4.